File: programmers/시저 암호.py
def solution(s, n):
    answer = ''
    for char in s:
        if char == ' ':
            answer += ' '
        else:
            if char.isupper():
                answer += chr((ord(char) - ord('A') + n) % 26 + ord('A'))
            else:
                answer += chr((ord(char) - ord('a') + n) % 26 + ord('a'))
    return answer

File: programmers/level1/시저암호.cpp
#include <string>

using namespace std;

string solution(string s, int n) {
    string answer = "";
    
    for (char c : s) {
        if (c >= 'A' && c <= 'Z') { // 대문자
            answer += 'A' + (c + n - 'A')%26;
        } else if (c >= 'a' && c <= 'z'){ // 소문자
            answer += 'a' + (c + n - 'a')%26;
        } else { // 공백
            answer += c;
        }
    }
    return answer;
}